src/scss/attendee/theme.scss

```diff
@@ -29,6 +29,9 @@
 
   --toggler-off-color: rgba(255, 255, 255, 0.3);
 
+  --button-disabled-background: rgb(183, 169, 217);
```

Comment:
[NOTE] Once `theme.scss` gets sufficiently large enough, we can analyze it and see if so many "colors" are needed (there are different alpha versions of the same color, as an example)